tldr/semantic.py

```
@@ -321,34 +374,50 @@ def extract_units_from_project(project_path: str, lang: str = "python", respect_
 
-        # Build call/called_by maps
-        calls_map = {}  # func -> [called functions]
-        called_by_map = {}  # func -> [calling functions]
+            # Build call/called_by maps
+            calls_map = {}  # func -> [called functions]
+            called_by_map = {}  # func -> [calling functions]
 
-        for edge in call_graph.edges:
-            src_file, src_func, dst_file, dst_func = edge
+            for edge in call_graph.edges:
+                src_file, src_func, dst_file, dst_func = edge
```

Ruff (0.14.13)

Unpacked variable `src_file` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)